reV/supply_curve/supply_curve.py

```diff
@@ -855,12 +855,6 @@ def full_sort(self, sort_on='total_lcoe',
                                        sort_on=sort_on, columns=columns,
                                        downwind=downwind)
 
-        if 'offshore' in supply_curve:
-            sum_cols = {'combined_cap_cost': ['array_cable_CAPEX',
-                                              'export_cable_CAPEX',
-                                              'trans_cap_cost_per_mw']}
```

Comment:
@MRossol fyi i think we talked about this previously but i remember what this was for. It was to sum up some of the cabling costs to get an idea of the offshore transmission capex. You can do most of this in NRWAL now and manually sum some columns in the SC table otherwise. I think it's safe to remove and the hard-coded column names were causing travis problems.

Reply:
Sounds good to me and glad Travis' bug wasn't "me"

Reply:
I wasnt blaming you but i think its good to remove the hard-coded col names nonetheless

Reply:
I wasn't saying you were, I was just worried i'd fucked up seeing as i pushed that at 10pm last night and the first thing I saw when logging on was, "it failed"...

Reply:
oh haha yeah gotcha, might have misread. yeah the tests are still failing but thats unrelated i think.